### PR TITLE
add option showHeader

### PR DESCRIPTION
**Pull request by vendeeglobe**
HTML Inline Renderer -> Options

Name            | Type         | Default             | Description
--------------- | ------------ | ------------------- | -----------
showHeader   | bool        | true | Generation of the table header.

https://github.com/JBlond/php-diff/wiki/2.-Parameters-and-Options

### CODE REVIEW

File: lib/jblond/Diff/Renderer/Html/SideBySide.php [hunk 1]
```diff
@@ -72,15 +74,22 @@ public function render()
      */
     public function generateDiffHeader(): string
     {
-        return <<<HTML
+        $html =  <<<HTML
```
Comment (JBlond) [inline lib/jblond/Diff/Renderer/Html/SideBySide.php:77]: This is not a clean solution.

**Comment by DigiLive** [comment]
JBlond wrote:
> I don't like this change.

I vote for what I offered at issue https://github.com/JBlond/php-diff/issues/53#issuecomment-662479284

IMHO overriding is the cleanest and most versatile way to generate custom output, based on a current renderer.
I'm not saying the request of vendeeglobe is invalid, but It's unmanageable for us to implement each and every requests for custom output.

However, I do appreciate his usage of our package and the feedback he gives us.
I'm willing to help him to implement the suggested overrides.
We can discuss it at https://github.com/JBlond/php-diff/discussions if he pleases so.

**Comment by JBlond** [comment]
@vendeeglobe is that an option for you that DigiLive helps you to make that changes in your repo?

**Comment by JBlond** [comment]
There wasn't any feedback within two weeks. Please add another commend, if you want to contiune.

**Pull request closed without merging**